src/routers/bookRouter.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const bookController = require('../controllers/bookController')
+
+const router = express.Router();
+
+router.get("/", bookController.getAllBooks);
+router.get("/:mainId", bookController.getBooksById);
```

Comment:
mainCategoryId를 받는 것이 명확하게 보일 수 있도록 수정해주세요.
```suggestion
router.get("/:mainCategoryId", bookController.getBooksByMainCategoryId);
```